apps/www/src/lib/apollo/cms-client.ts

```diff
@@ -25,6 +25,7 @@ export function getCMSClient(): NextSSRApolloClient<NormalizedCacheObject> {
         Authorization: `${process.env.DATO_CMS_API_TOKEN}`,
         // forbid invalid content to allow strict type checking
         'X-Exclude-Invalid': 'true',
+        'X-Environment': process.env.DATO_CMS_ENVIRONMENT,
```

Comment:
Let's add a check for the DATO_CMS_ENVIRONMENT env var too. If it's `undefined`, the query will fail.

Reply:
Alternatively, just fall back to `'main'`